frontend_tests/node_tests/people.js

```diff
@@ -244,38 +372,13 @@ run_test('user_timezone', () => {
 });
 
 run_test('user_type', () => {
-    const realm_admin = {
-        email: 'realm_admin@example.com',
-        full_name: 'Realm Admin',
-        user_id: 32,
-        is_admin: true,
-        is_guest: false,
-        is_bot: false,
-    };
-    const guest = {
-        email: 'guest@example.com',
-        full_name: 'Guest User',
-        user_id: 33,
-        is_admin: false,
-        is_guest: true,
-        is_bot: false,
-    };
-    const bot = {
-        email: 'bot@example.com',
-        full_name: 'Example Bot',
-        user_id: 34,
-        is_admin: false,
-        is_guest: false,
-        is_bot: true,
-    };
-
     people.add(realm_admin);
     people.add(guest);
-    people.add(bot);
+    people.add(bot_botson);
```

Comment:
We already add `bot_botson` in the previous test (line 346) and haven't reinitialized `people.js` after that.

Obviously, there isn't any harm in re-adding someone, but I'm wondering if it makes sense to reset the people data after each test and have each test be more self-contained.

Reply:
Fixed.  We should probably just wrap `run_test` to always clear `people` with a two-liner, but that's a separate cleanup project.